cmd/server/main.go

```diff
@@ -13,6 +13,7 @@ import (
 )
 
 func run(reg *descriptor.Registry, swaggerFiles []string) error {
+	fmt.Printf("Atlas openapiv2 patch - Input args: %+v\n", reg)
```

Comment:
I would prefer to keep this output as this helps to know what was used as args.

Reply:
Then you should use `glog.V(1).Infof()` as done in the rest of the file.

Reply:
I had tried  glog.V(1).Infof() but it didn't dump the output in my run inside Goland.  It might need the info level to be set for the UT.  I prefer to keep it this way and it is probably okay since this is printed only in the beginning.

Reply:
I don't know how glog works, you might want to check the documentation: https://github.com/golang/glog#glog. This is the existing logging mechanism and should be used by new pieces of code. The fact it doesn't work in your IDE is not a good reason to not use it and use something else. Also in production this module isn't going to be used in an IDE but from the command line.

Reply:
I had tested earlier with the glog by bringing a version into atlas-gentool and running the tool for ipamsvc and it didn't print out the cmd line arguments.  May be it might if pass the -v arg that glog needs in our make file but I would prefer the tool to always print out the arguments that it uses so that we can know if the right tool and args are being used for generation.  If you still prefer to use the glog way, i would rather drop this code as it will not be used by the multiple repo chain of things needed to get that info.  What is your preference here?

Reply:
You need to use something like `-logtostderr=true -v=1` otherwise the logs aren't dumped to the terminal.

If the logged messages could be useful in production, `glog` must be used to conform with the existing code. If they are only useful for your debugging, they can be removed.

Reply:
Sure. updated